tests/installed/itest-payload-link.sh

```diff
@@ -57,29 +56,25 @@ fi
 mkfs.xfs -m reflink=1 ${blkdev}
 
 mount ${blkdev} mnt
+cd mnt
 
-test_tmpdir=$(pwd)/mnt
-cd ${test_tmpdir}
-
+# Test that coreutils will do reflink
```

Comment:
Also I changed this to just require that reflinks work since that's the point of the test.